mobileApp/App.js

```diff
@@ -1,5 +1,5 @@
-import React, { Component } from "react";
-import { StyleSheet, Text, View } from "react-native";
+import React, {Component} from "react";
+import {StyleSheet, Text, View} from "react-native";
```

Comment:
Why have you changed this? What linter are you using?

Reply:
Reverted changes